mesonbuild/compilers/compilers.py

```diff
@@ -133,6 +139,12 @@ def is_library(fname):
                                  'release': ['-Wl,-O1'],
                                  'minsize': [],
                                  }
+arm_buildtype_linker_args = {'plain': [],
+                             'debug': [],
+                             'debugoptimized': [],
+                             'release': [],
+                             'minsize': [],
+                            }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
Agreed. Will submit new patch change to reflect these changes